configuration/pih/htmlforms/hiv/section-hiv-state.xml

```diff
@@ -156,10 +156,248 @@
                 jq("#editChwNameDiv").hide();
             }
           });
+
+    });
+</script>
+</excludeIf>
+
+<!-- This method allows you to retrieve the last date of completion of izoniazid -->
+<excludeIf velocityTest="$fn.getObs($encounter, 'PIH:14535')">
+<script type="text/javascript">
+
+    jq(document).ready(function() {
+        
+
+        const contextPath = window.location.href.split('/')[3];
+        const apiBaseUrl =  "/" + contextPath + "/ws/rest/v1";
+        const patientUuid = '<lookup expression="patient.uuid"/>';
+
+        let currentEncounterDate = new Date();
```

Comment:
Current encounter date shouldn't default to "today" if the form field isn't present, but rather it should default to the current encounter date of the existing encounter (if any) via velocity expression.